src/_mock/_app.ts

```diff
@@ -1,52 +0,0 @@
-import { noCase } from 'change-case';
```

Comment:
Why removing all mocking date.. can this be used for tests later?

Reply:
It can be added if needed from the template. For now, we keep just the files and components we need.